### PR TITLE
[QMR] 508 Remediation - Labels for Sensory Characters

### PR DESCRIPTION
**Pull request by ajaitasaini**
### Description

Added tooltip for icons in QMR

### Related ticket(s)

MDCT-1949

---
### How to test

- Login
- - Confirm new AX functionality: 
    - Three icons were affected: More Actions, Delete, and Date Picker -- all should have a tooltip when hovering 
    - When tabbing through months in the Date Picker, voiceover should now read the year along with the month. e.g.:"May 2022, June 2022, etc"
- More actions icon can be found on the main page
- Delete icon can be found under adding another category under OMS
- Date Picker is found by clicking "No'' on a date range question

### Important updates

---
### Author checklist

- [ ] I have performed a self-review of my code
- [ ] I have added [thorough](https://bit.ly/3zPrxuZ) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

File: services/ui-src/src/components/MonthPicker/calendarPopup.tsx [hunk 1]
```diff
@@ -118,7 +120,7 @@ export const MonthPickerCalendar = ({
               return (
                 <CUI.Button
                   key={`${value}-${index}`}
-                  aria-label={value}
+                  aria-label={value + " " + year}
```
Comment (gmrabian) [inline services/ui-src/src/components/MonthPicker/calendarPopup.tsx:123]: nit: could write this a little cleaner with backticks, like on the line above

\`${value} ${year}\`

**Review by gmrabian — COMMENTED** [review]
Not sure how this could have happened from your changes but I noticed the option to add more categories didn't display properly. Instead of 'Race' it displayed this random id

Screenshot 2023-08-30 at 12 47 15 PM

All the tool tips look good! Again, not sure how this happened or if it's related.

**Comment by ajaitasaini** [comment]
> Not sure how this could have happened from your changes but I noticed the option to add more categories didn't display properly. Instead of 'Race' it displayed this random id
> 
> Screenshot 2023-08-30 at 12 47 15 PM
> 
> All the tool tips look good! Again, not sure how this happened or if it's related.

@gmrabian thanks for catching this! it was unrelated + fixed last week, but i just forgot to merge :')

File: services/ui-src/src/components/MonthPicker/calendarPopup.tsx (hunk 1 above)
Comment (ajaitasaini) [reply services/ui-src/src/components/MonthPicker/calendarPopup.tsx]: makes sense, ty!!

**Review by gmrabian — DISMISSED** [review]
looks good!

**Review by keeysnc — APPROVED** [review]

**Review by benmartin-coforma — DISMISSED** [review]

**Review by BearHanded — APPROVED** [review]

**Review by karla-vm — APPROVED** [review]

**Pull request merged**